flocker/node/_loop.py

```diff
@@ -53,6 +53,10 @@
 )
 from ..control._persistence import to_unserialized_json
 
+# The maximum number of seconds an agent will wait before attempting to
+# reconnect to the control service.
+MAXIMUM_RECONNECT_DELAY = 30
```

Comment:
Consider making this a command line parameter. Seems reasonable that an end-user would want to specify this value.